crates/sui-core/src/authority_active/checkpoint_driver/mod.rs

```diff
@@ -85,10 +85,10 @@ impl Default for CheckpointProcessControl {
         CheckpointProcessControl {
             delay_on_quorum_failure: Duration::from_secs(10),
             delay_on_local_failure: Duration::from_secs(3),
-            long_pause_between_checkpoints: Duration::from_secs(60),
+            long_pause_between_checkpoints: Duration::from_secs(120),
             timeout_until_quorum: Duration::from_secs(60),
             extra_time_after_quorum: Duration::from_millis(200),
-            consensus_delay_estimate: Duration::from_secs(1),
+            consensus_delay_estimate: Duration::from_secs(2),
```

Comment:
I honestly think we should have a delay estimation as in PR https://github.com/MystenLabs/sui/pull/3548